### ISSUE TITLE
feature request: allow changing branch name and PR description

### ISSUE THREAD

**Issue by IamShobe** [ilaif/goplicate#17]
Hi there :),
A feature request that I think will be useful:
Allow changing branch name and PR Description that opens automatically:
I would imagine something like:
```
goplicate sync --base main --publish --branch "chore/something" --message "some message"
```
if you don't have the --message flag an editor (`$EDITOR` / `editor` binary) opens up automatically (like `git commit`) - which the base template already exists.
message should be asked only once (not every repo).

**Comment by ilaif** [comment]
Hi @IamShobe,
Please see https://github.com/ilaif/goplicate/pull/19 🤩

**Comment by ilaif** [comment]
Resolved in #19
